torch/_subclasses/fake_tensor.py

```diff
@@ -513,7 +518,27 @@ def index_put_(fake_mode, func, *args, **kwargs):
     return new_kwargs["input"]
 
 
-@register_op_impl(lambda fn: fn in _device_not_kwarg_ops)
+@register_op_impl(lambda fn: fn in _nested_constructors)
+def nested_constructor(fake_mode, func, *args, **kwargs):
+    assert func not in _non_kwarg_device_constructors
+    new_args, new_kwargs = normalize_function(
```

Comment:
Can you use `normalize_to_only_use_kwargs=False` ?  if you invoke this with all kwargs we are going to access `new_args[3]` below and it will be out of bounds.

Reply:
I'm 99% sure `normalize_to_only_use_kwargs=False` does not mean normalize to use args when possible. using kwargs in a test would check

Reply:
Yes